values.yaml

```diff
@@ -281,7 +281,8 @@ persistence:
     gcs:
       bucket: bucketname
       # The base64 encoded json file which contains the key
-      encodedkey: base64-encoded-json-key-file
+      # If encodedkey is unset and GCS enabled it defaults to service account authentication for workload identity
```

Comment:
Please also update the `readme.md` for this property

Reply:
I am not sure on the best way to add it to the README since it is a sub value of persistence.imageChartStorage.type that is already covered with 1 line for all the types this is why I added a comment in the values file for it. Thoughts?

`| `persistence.imageChartStorage.type`                                        | The type of storage for images and charts: `filesystem`, `azure`, `gcs`, `s3`, `swift` or `oss`. The type must be `filesystem` if you want to use persistent volumes for registry and chartmuseum. Refer to the [guide](https://github.com/docker/distribution/blob/master/docs/configuration.md#storage) for more information about the detail `